Jenkinsfile

```diff
@@ -105,15 +105,20 @@ pipeline {
         }
       }
       steps {
+        sh 'make setup-git'
+        sh 'git fetch --all'
         unstash 'psammead-publishes'
-        sh 'make bump-dependants'
+        sh 'make talos'
       }
       post {
         always {
           script {
             stageName = env.STAGE_NAME
           }
         }
+        cleanup {
+          sh 'chmod -R 777 .git'
```

Comment:
Needed because there were issues where the `.git` folder was changed (via one of the git commands), which meant when the pipeline next ran, when jenkins tried to check out the branch, it couldnt as it didnt have write access. 777 is a 'give everything back to jenkins' permision level :P

Reply:
Does this not work with `755` doing a chmod to 777 is never a good thing.

Reply:
I dont think `755` will work as that doesnt allow public write access, which i think we need to allow jenkins to checkout and change the .git folder

Reply:
This is OK to get going but we need a permanent solution that doesn't grant global write permission. There must be another way to allow subsequent runs to overwrite existing files, e.g. have we checked the owner/group of these files? Please can you create an issue to revisit?